frappe/public/scss/common/indicator.scss

```diff
@@ -3,14 +3,14 @@
 	color: var(--text-on-#{$color});
 	&::before,
 	&::after {
-		background: var(--text-on-#{$color});
+		background: var(--bg-#{$color});
 	}
 }
 
 @mixin indicator-color($color) {
 	&::before,
 	&::after {
-		background: var(--text-on-#{$color});
+		background: var(--bg-#{$color});
```

Comment:
This will fix it for dark mode but now light mode will be broken

Reply:
On my trail edit light mode works also. Do you have a different result ?